crl/validator_test.go

```diff
@@ -142,12 +146,27 @@ func TestValidator_IsRevoked(t *testing.T) {
 		assert.True(t, crlValidator.IsSynced(0))
 	})
 
-	t.Run("should return false if the certificate was not revoked even though the bit was set", func(t *testing.T) {
-		t.Parallel()
+	t.Run("should return false if the crl is expired", func(t *testing.T) {
+		oldNowFunc := nowFunc
+		nowFunc = func() time.Time {
```

Comment:
I always wonder what happens at the end of the function, should you restore `nowFunc` in a defer? Or are package-level variables re-initialized for every test?

Reply:
For subtests you need to restore it, for complete different test package, I don't know.